File: desafio-02/veduardo93/primos.cpp
#include <iostream>

int main() {
    for (int i = 1; i <= 10000; i++) {
        bool primo = true;

        for (int j = 2; j <= i / 2 && primo; j++)
            if (i % j == 0)
                primo = false;
        if (primo)
            std::cout << i << " ";
    }
}

File: desafio-02/roberto0arruda/python/primos.py
"""
Autor: Roberto Arruda
#02 - Primos
"""
for num in range(1, 10000 + 1):
  for n in range(2, num - 1):
    if (num % n == 0):
      break
  else:
    print("{} é primo".format(num), end='\n')
